packages/SwingSet/src/vats/vat-admin/vat-vat-admin.js

```diff
@@ -398,7 +391,7 @@ export function buildRootObject(vatPowers, _vatParameters, baggage) {
     // reject unknown options
     const unknown = Object.keys(rest).join(',');
     if (unknown) {
-      assert.fail(`CreateVatOptions: unknown options ${unknown}`);
+      Fail`CreateVatOptions: unknown options ${q(unknown)}`;
```

Comment:
I'm going to trust you on these `q(…)` introductions.

Reply:
I'm pretty sure @erights will dispute this (and it's not worth fighting about) but I'd change the default to not censor and instead have something like a c(...) function to censor.   IMHO the default censorship of strings has never provided much value and has cost a lot in terms of debugging.  Another minor thing that's disruptive is that `q` outputs its arg in quotation marks, which breaks tests and also inhibits one's ability to control the format of output (at some point it might be useful to add something that performs the uncensoring mission of `q` but doesn't insert the quote characters, but it's not the least bit pressing).